src/ui/components/Icon/styles.scss

```diff
@@ -8,18 +8,22 @@
 }
 
 .Icon-magnifying-glass {
-  background: url('~amo/img/icons/magnifying-glass.svg') 0 50% no-repeat;
+  background: url('~amo/img/icons/magnifying-glass.svg') center no-repeat;
   background-size: cover;
 }
 
 .Icon-eye {
-  background: url('./eye.svg') 0 50% no-repeat;
+  background: url('./eye.svg') center no-repeat;
   background-size: cover;
   width: 23px;
 }
 
 .Icon-user {
-  background: url('./user.svg') 0 50% no-repeat;
+  background: url('./user.svg') center no-repeat;
+  background-size: contain;
+}
+
+.Icon-user-dark {
+  background: url('./user-dark.svg') center no-repeat;
```

Comment:
Yeah, I started on it but it was too much work to convert old icons (at the time). It's a great lazy Sunday patch that I'm hoping to do in the future; CSS control over SVG is like one of the best reasons to use them :-)